.gitlab-ci.yml

```
@@ -24,8 +24,8 @@ mirror-from-github:
     - cd $DIR/qem-dashboard
     - LOCAL_CHECKOUT=$(git rev-parse HEAD)
     - >- 
-      [[ $LOCAL_CHECKOUT -eq $GIT_COMMIT_SHA ]] && exit 0
-    - git fetch --unshallow main
+      [[ "$LOCAL_CHECKOUT" == "$GIT_COMMIT_SHA" ]] && exit 0
```

quotes on the left hand side are not needed
```suggestion
      [[ $LOCAL_CHECKOUT == "$GIT_COMMIT_SHA" ]] && exit 0
```